docs/orchestrator.md

```diff
@@ -236,6 +236,8 @@ celestia-appd query staking validator <validator_valoper_address>
 
 Now, you can restart the orchestrator, and it should start signing.
 
+Note: A validator set change is triggered if more than 5% of the total staking power of the network changes (0.5% for BSR). This means that even if you change your EVM address, and you don't see your orchestrator signing, it's alright. Just wait until the validator set changes, and then your orchestrator will automatically start signing.
```

Comment:
I might simply mention that this could ber different per network, or explain BSR here since out of context it might be confusing

Reply:
I put here BSR because first validators who will read this will be from BSR. We can update it later. What do you think?